go.mod

```diff
@@ -231,7 +228,3 @@ require (
 // gopacket/gopacket with:
 // - https://github.com/gopacket/gopacket/pull/101
 replace github.com/gopacket/gopacket => github.com/gopacket/gopacket v1.3.2-0.20241202175635-b43272ae1eb8
-
-replace k8s.io/component-base => k8s.io/component-base v0.0.0-20240417101527-62c04b35eff6
-
-replace oras.land/oras-go/v2 => oras.land/oras-go/v2 v2.0.0-20240123101058-64fedf45bfd3
```

Comment:
For the record, it was initially needed for commit 16b44f58ffb86d626a59b004f4d4d907af0ccff1 but this is no longer needed because https://github.com/oras-project/oras-go/pull/647 is in the oras-go release.